lib/datadog/grok/patterns/java.pattern

```diff
@@ -15,5 +15,5 @@ CATALINA_DATESTAMP %{MONTH} %{MONTHDAY}, 20%{YEAR} %{HOUR}:?%{MINUTE}(?::?%{SECO
 # yyyy-MM-dd HH:mm:ss,SSS ZZZ eg: 2014-01-09 17:32:25,527 -0800
 TOMCAT_DATESTAMP 20%{YEAR}-%{MONTHNUM}-%{MONTHDAY} %{HOUR}:?%{MINUTE}(?::?%{SECOND}) %{ISO8601_TIMEZONE}
```

Comment:
Fwiw, I did stop flagging/correcting items in the pattern files, as they are mostly patterns.

It's possible these are third party items in which case I can eventually send a PR to the source if it's identified.